druid-shell/src/platform/x11/window.rs

```diff
@@ -220,6 +221,24 @@ impl WindowBuilder {
         let screen_num = self.app.screen_num();
         let id = conn.generate_id()?;
         let setup = conn.setup();
+
+        let scale = match self.app.rdb.get_value::<f64>("Xft.dpi", "") {
```

Comment:
I agree that this is the right way to do it, but could we have an environment variable override also? They're handy for testing...